### ISSUE TITLE
Top-level orbax import 0.5.* globally breaks logging

### ISSUE THREAD

**Issue by qGentry** [google/orbax#699]
Importing orbax 0.5.* (tested on 0.5.0, 0.5.1, 0.5.2. On 0.4.3 there is no such problem) somehow (i didn't even know that was possible) prevents any logging configuration after import.

```python3
import logging

logger = logging.getLogger(__name__)

import orbax.checkpoint as ocp

logging.basicConfig(
    format="%(asctime)s %(levelname)-8s %(message)s",
    level=logging.INFO,
    datefmt="%Y-%m-%d %H:%M:%S",
)
logging.info("Some info")
logging.warning("Some warning")
```

```bash
root@ffisin-dev:/mnt/llm/home/ffisin/papyrax# python3 test2.py 
WARNING:root:Some warning
```
Logging configuration is completely ignored. Both format and level.

```python3
import logging

logger = logging.getLogger(__name__)

# import orbax.checkpoint as ocp # <- commenting orbax import

logging.basicConfig(
    format="%(asctime)s %(levelname)-8s %(message)s",
    level=logging.INFO,
    datefmt="%Y-%m-%d %H:%M:%S",
)
logging.info("Some info")
logging.warning("Some warning")
```
After commenting orbax import everything is ok.
```bash
root@ffisin-dev:/mnt/llm/home/ffisin/papyrax# python3 test2.py 
2024-02-01 19:27:39 INFO     Some info
2024-02-01 19:27:39 WARNING  Some warning
```

**Comment by qGentry** [comment]
One more thing:
Importing orbax AFTER logging configuration is done seems to fix this problem
```
import logging

logger = logging.getLogger(__name__)


logging.basicConfig(
    format="%(asctime)s %(levelname)-8s %(message)s",
    level=logging.INFO,
    datefmt="%Y-%m-%d %H:%M:%S",
)
import orbax.checkpoint as ocp
logging.info("Some info")
logging.warning("Some warning")
```

```
root@ffisin-dev:/mnt/llm/home/ffisin/papyrax# python3 test2.py 
2024-02-01 19:36:56 INFO     Failed to apply nest_asyncio. Received error: no running event loop # < this is log from orbax itself
2024-02-01 19:36:56 INFO     Some info
2024-02-01 19:36:56 WARNING  Some warning
```

**Comment by niketkumar** [comment]
Hi @qGentry, Apologies for the inconvenience and thank you for reporting the issue!

The issue is resolved in `0.5.3`. Please let me know if you are still facing the issue.

**Comment by qGentry** [comment]
just checked it - now it works as expected, thank you!
